org.sonatype.m2e.egit/src/org/sonatype/m2e/egit/internal/EgitScmHandler.java

```diff
@@ -101,6 +101,28 @@ protected String normalizeUri(String uri) throws URISyntaxException {
       throw new URISyntaxException(uri, "Invalid git URI");
     }
 
+    // Normalize uri like
+    // git@github.com:errai/errai.git/errai-common
+    // into
+    // git://github.com/errai/errai.git
+    // These 3 distinctions may happen independently in other cases.
+    //1. Replace @ with ://
+    if(uri.startsWith("git@")) {
+      uri = "git://" + uri.substring(4);
+    }
+    //2. Replace github.com: with github.com/
+    String githubDotCom = "github.com";
```

Comment:
The host might be arbitrary, like bitbucket.org: https://bitbucket.org/cofarrell/bower-maven-plugin/src/ee0f41a83905?at=master